src/main/groovy/io/openliberty/tools/gradle/tasks/DevTask.groovy

```diff
@@ -500,13 +499,23 @@ class DevTask extends AbstractFeatureTask {
         @Override
         public boolean updateArtifactPaths(ProjectModule projectModule, boolean redeployCheck, boolean generateFeatures, ThreadPoolExecutor executor)
                 throws PluginExecutionException {
-            // not supported for Gradle, only used for multi module Maven projects
+            // unable to identify the changes made, showing option for user. always return false because action is invoked manually
+            if (isMultiModuleProject()) {
```

Comment:
Where did this boolean get set?

Reply:
Its set in DevUtil( already there as part of ci maven multi module changes. we are passing projectModuleList created by getProjectModules() as upstreamProjects)

![image](https://github.com/user-attachments/assets/3ba6c8f3-714f-4d15-9013-c3f09e0395c5)


![image](https://github.com/user-attachments/assets/8b77df93-31f1-4d17-bfea-64a249ba4d0d)